Ported/combat-bees/Assets/Scripts/Jobs/SpawnJob.cs

```diff
@@ -30,7 +30,7 @@ public void Execute(int index)
 
         float3 position = Aabb.Center + Aabb.Extents * randomf3;
 
-        ECB.AddComponent(index, entity, new LocalToWorldTransform{Value = UniformScaleTransform.FromPositionRotationScale(position, InitTransform.Value.Rotation, InitTransform.Value.Scale)});
-        ECB.AddComponent(index, entity, new Faction{Value = InitFaction});
+        ECB.SetComponent(index, entity, new LocalToWorldTransform{Value = UniformScaleTransform.FromPositionRotationScale(position, InitTransform.Value.Rotation, InitTransform.Value.Scale)});
+        ECB.SetComponent(index, entity, new Faction{Value = InitFaction});
```

Comment:
Does food need a faction component?  Locally I made a similar change and it looks more like this
```suggestion
        if (InitFaction == (int)Factions.None)
            ECB.SetComponent(index, entity, new Faction { Value = InitFaction });
```
There is no component that identifies "food/resource" right now, but I'm not sure if Factions.None is the best identifier. It's likely we want a food tag, unless we want to give bees and food a faction shared component, which I think is reasonable because it would make a target selection easier (Query.ToEntityArray, pick a random value in the array for all bees selecting a new target).  If that's the case Factions.None is not really "Factions.None" it's Factions.Food